### PR TITLE
[bugfix] geohash lat/long is reversed

### PR DESCRIPTION
**Pull request by mistercrunch**
This allows support for reversing geohashes. Note that the default option was the wrong way.
screen shot 2018-08-21 at 8 43 36 pm

### CODE REVIEW

File: superset/viz.py [hunk 1]
```diff
@@ -2113,17 +2121,13 @@ def process_spatial_data_obj(self, key, df):
         elif spatial.get('type') == 'delimited':
             lon_lat_col = spatial.get('lonlatCol')
             df[key] = df[lon_lat_col].apply(self.parse_coordinates)
-
             if spatial.get('reverseCheckbox'):
-                df[key] = [
-                    tuple(reversed(o)) if isinstance(o, (list, tuple)) else (0, 0)
```
Comment (betodealmeida) [inline superset/viz.py]: I'm happy we're getting rid of the `(0, 0)` here.

File: superset/viz.py [hunk 2]
```diff
@@ -2113,17 +2121,13 @@ def process_spatial_data_obj(self, key, df):
...
-                ]
+                reverse_latlong()
             del df[lon_lat_col]
         elif spatial.get('type') == 'geohash':
-            latlong = df[spatial.get('geohashCol')].map(geohash.decode)
-            df[key] = list(zip(latlong.apply(lambda x: x[0]),
-                               latlong.apply(lambda x: x[1])))
+            df[key] = df[spatial.get('geohashCol')].map(geohash.decode)
+            if not spatial.get('reverseCheckbox'):
+                reverse_latlong()
```
Comment (betodealmeida) [inline superset/viz.py]: This `if` block is super confusing... can't we fix `geohash.decode` instead?

File: superset/viz.py (hunk 1 above)
Comment (mistercrunch) [reply superset/viz.py]: yeah that place on the west coast of africa that throws the autozoom off...

File: superset/viz.py (hunk 2 above)
Comment (mistercrunch) [reply superset/viz.py]: I can wrap it in a lambda that reverses it

**Review by betodealmeida — APPROVED** [review]
Gogogo

**Pull request merged**